docs/rules/0148/use-uid.md

```diff
@@ -0,0 +1,63 @@
+---
+rule:
+  aip: 148
+  name: [core, '0148', use-uid]
+  summary: Use uid instead of id in resource messages.
+permalink: /148/use-uid
+redirect_from:
+  - /0148/use-uid
+---
+
+# Human names
+
+This rule encourages the use of `uid` instead of `id` for resource messages, as
+mandated in [AIP-148][].
+
+## Details
+
+This rule looks for fields named `first_name` and `last_name`, and complains if
+it finds them, suggesting the use of `given_name` and `family_name`
+(respectively) instead.
+
+## Examples
+
+**Incorrect** code for this rule:
+
+```proto
+// Incorrect.
+message Human {
+  string first_name = 1;  // Should be `given_name`.
+  string last_name = 2;   // Should be `family_name`
+}
+```
+
+**Correct** code for this rule:
+
+```proto
+// Correct.
+message Human {
+  string given_name = 1;
+  string family_name = 2;
+}
+```
+
+## Disabling
+
+If you need to violate this rule, use a leading comment above the field or its
+enclosing message. Remember to also include an [aip.dev/not-precedent][]
+comment explaining why.
+
+```proto
+// (-- api-linter: core::0148::human-names=disabled
```

Comment:
change api linter example?

side note: I think we could generate a ton of this documentation with the right metadata.

Reply:
Fixed